.drone.yml

```
@@ -4,24 +4,24 @@ type: docker
 name: test
 steps:
 - name: install
-  image: danlynn/ember-cli:3.20.0
+  image: danlynn/ember-cli:3.28.5
```

```suggestion
  image: danlynn/ember-cli:3.28.5-node_16.13
```

I would suggest using node 16 here as well